### PR TITLE
Limit pytest version to <3.2

### PR DESCRIPTION
**Pull request by larrybradley**

### CODE REVIEW

File: .run_docker_tests.sh [hunk 1]
```diff
@@ -16,6 +16,9 @@ uname -m
 echo "Output of sys.maxsize in Python:"
 python -c 'import sys; print(sys.maxsize)'
 
+# temporary
```
Comment (eteq) [inline .run_docker_tests.sh]: Maybe specify the conditions of temporary?  Just so that someone else might know when to remove it...?

**Comment by eteq** [comment]
Just to clarify, is this is to correct the issue from https://github.com/astropy/photutils/pull/578#issuecomment-319799878 ?

**Comment by bsipocz** [comment]
@eteq - Yes, this a workaround for that. On the long term we either need to require pytest 3.1+, or remove the usage of pytest.mark within a parametrize decorator as it was suggested here: https://github.com/astropy/astropy/pull/6419#issuecomment-319977438

**Comment by bsipocz** [comment]
@eteq - Oh, for *that*, the workaround is to limit the version in ci-helpers (almost done, but I'm strugling with appveyor  bit), and build a pytest 3.1 version for py3.4 in the astropy-ci-extras channel

**Pull request merged**